src/main/java/com/revature/studyforce/flashcard/model/Vote.java

```diff
@@ -0,0 +1,51 @@
+package com.revature.studyforce.flashcard.model;
+
+import com.revature.studyforce.user.model.User;
+import lombok.*;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Model to store each vote on submitted flashcard answers
+ * @author Elizabeth Ye
+ */
+@Entity
+@Table(name = "answer_vote")
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@AllArgsConstructor
+@NoArgsConstructor
+public class Vote {
+
+    /**
+     * Serial primary key voteId for Vote object
```

Comment:
class properties should not be documented one by one like this. Instead, simply use good naming convention to avoid ambiguity.